_Sidebar.md

```diff
@@ -15,15 +15,15 @@
 12. **[[Interoperability|Interoperability]]**
 13. **[[Kokkos and Virtual Functions|Kokkos and Virtual Functions]]**
 ### **[[Content: Algorithms|ContentAlgorithms]]**
-1. **[[Sorting|Sorting]]**
-2. **[[Random Numbers|Random-Number-Generators]]**
-3. **[[Iterators|Iterators]]**
-4. **[[Std Algorithms|StdAlgorithms]]**
+1. **[[Random Numbers|Random-Number-Generators]]**
+2. **[[Iterators|Iterators]]**
+3. **[[Std Algorithms|StdAlgorithms]]**
    1. **[[NonModSequenceOps|StdNonModSeq]]**
    2. **[[ModSeqOps|StdModSeq]]**
    3. **[[PartioningOps|StdPartioningOps]]**
    4. **[[Numeric|StdNumeric]]**
    5. **[[StdMinMaxElement|StdMinMaxElement]]**
+   6. **[[Sorting|StdSorting]]**
```

Comment:
Why do you think it should be moved there?
It could introduce some confusion such as:  Why is `Kokkos::sort` not in `Experimental::` like the rest of the algorithms?  Why does it not have overloads that take iterators like all other algorithms?

Reply:
Nevermind these are `Kokkos::Experimental::is_sorted[_until]`